kubernetes/env-create.md

```diff
@@ -0,0 +1,232 @@
+---
+category: Kubernetes
+expires: 2018-01-31
+---
+# Creating a Kubernetes Environment
+
+## Aim
+
+This is a guide to creating a non-production environment in our non-production Kubernetes cluster. This document is intended for any developers who are interested in creating a development or staging environments within Kubernetes.
+
+## Description
+
+[K8s-nonprod-environments repo: - https://github.com/ministryofjustice/k8s-nonprod-environments](https://github.com/ministryofjustice/k8s-nonprod-environments)
+
+This repo contains the necessary files to create a pipeline in aws to create Kubernetes cluster namespaces and resources after a push has been made to the master branch of this repo.
+
+This repo contains the following files and directories
+
+#### Terraform
+
+This directory contains terraform resources to create AWS pipeline for creation of kubernetes namespave and resource creation.
+
+#### Buildspec
+
+The buildspec.yaml file contains codebuild specification that will create resources defined in the namespaces.
+
+#### Namespace
+
+The namespaces contains subdirectories named after each of the desired namespaces you want to create.
+
+You will be working within the namespace directory to create your environment.
+
+### GitHub directory structure
+
+#### k8s-non-prod-environment
+
+![Image](images/image4.png)
+
+This is the root of the repo, containing Terraform and Namespace directory
+
+#### /namespaces
+
+![Image](images/image5.png)
+
+This is the Namespace directory, this is where you will add create a directory for your service in the format $servicename-$env   
```

Comment:
You can put in line code into backticks (`code`) to make it easier to tell that it is code. So here you could change $servicename-$env to `$servicename-$env` to make it a little easier to read. 

There's a couple other example of this with the `$variable` in the main text that you might want to change.